symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkClientConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.symphony.bdk.core.config.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BdkClientConfig {
+
+    private static final String DEFAULT_SCHEME = "https";
+    private static final int DEFAULT_HTTPS_PORT = 443;
+
+    private String scheme = DEFAULT_SCHEME;
+    private String url;
```

Comment:
`host` sounds better than `url` WDYT? 

@symphony-youri actually most the time developers will only have to configure `host` and `port` for on-prem components, for on-cloud components it's even simpler as they would only have to configure the `host` (e.g. `devx1.symphony.com` or `corporate.symphony.com`)